autogpt_platform/frontend/src/app/layout.tsx

```diff
@@ -25,6 +27,8 @@ export default async function RootLayout({
 }>) {
   const headersList = await headers();
   const host = headersList.get("host") || "";
+  const previewStealingDev = environment.getPreviewStealingDev();
+  const isDev = environment.getAppEnv() === AppEnv.DEV;
```

Comment:
I think you have introduced an `isDev` function for this? is this different?

Reply:
Good call, there is an `isDev`, but annoyingly checks for the development build 😆 , but agree this would be nicer as an `is...` function 💅🏽